test/testcne/meson.build

```diff
@@ -128,6 +128,7 @@ test_names = [
     'log',
     'mbuf',
     'mempool',
+    'metrics',
```

Comment:
Please do not do this. We don't run the meson tests as root, and this test willl fail if it is not run as root. See #75